src/pages/doi.org/[...doi].tsx

```diff
@@ -679,7 +696,7 @@ const WorkPage: React.FunctionComponent<Props> = ({ doi, metadata, isBot = false
       <TitleComponent title={ReactHtmlParser(titleHtml)} titleLink={handleUrl} link={'https://doi.org/' + work.doi} rights={work.rights} />
            
       {content()}
-      {!isBot && relatedContent()}
```

Comment:
Did the isBot approach not work?
I see it has been removed in all rendering contexts